Software/Python/ir_remote_control/gobox_ir_receiver_libs/ir_debug_logger.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python 
```

Comment:
The comment was meant for the whole file, not the shebang. :) I know about the shebang.  I just put the comment at the top, because it concerned everything. Sorry for being unclear here.